### PR TITLE
Update travis ci

### PR DESCRIPTION
**Pull request by ljiang1**

### CODE REVIEW

**Comment by shakeelmohamed** [comment]
@ljiang1 change look good. If you click on details (or the red X) you'll see the Travis CI build status. We should file bugs internally for all test failures

File: .travis.yml [hunk 1]
```diff
@@ -30,8 +30,8 @@ before_install:
     - mkdir -p $SPLUNK_HOME/var/log/splunk
 
 env:
-    - SPLUNK_VERSION=6.4-sdk
-    - SPLUNK_VERSION=6.5-sdk
+    - SPLUNK_VERSION=6.6.5-sdk
```
[the remark this answers is not in the document]
Comment (ljiang1) [reply .travis.yml]: @shakeelmohamed this build ( getting splunk part)  seems to work fine without the last .  you think we still need that last .?

File: tests/test_service.py [hunk 2]
```diff
@@ -177,11 +179,12 @@ def setUp(self):
         self.opts = testlib.parse([], {}, ".splunkrc")
         self.service = client.Service(**self.opts.kwargs)
 
-        # Skip these tests if running below Splunk 6.2, cookie-auth didn't exist before
-        splver = self.service.splunk_version
-        # TODO: Workaround the fact that skipTest is not defined by unittest2.TestCase
-        if splver[:2] < (6, 2):
-            self.skipTest("Skipping cookie-auth tests, running in %d.%d.%d, this feature was added in 6.2+" % splver)
+        # remove these code as 6.2- is deprecated and splunk_version can't be got without login on splunk 6.6+
```
Comment (shakeelmohamed) [inline tests/test_service.py]: let's just remove the lines

File: tests/test_service.py [hunk 3]
```diff
@@ -158,12 +158,14 @@ def test_query_without_login_raises_http_401(self):
...
             else:
                 raise
-    
-    def test_server_info_without_login(self):
-        service = self._create_unauthenticated_service()
-        # Should succeed without AuthenticationError
-        service.info['version']
-    
+
+    # remove test as it is not supported on splunk 6.6+
```
Comment (shakeelmohamed) [inline tests/test_service.py]: let's remove it

**Pull request merged**